gcp-jobs/update-legal-filings/devops/vaults.gcp.env

```diff
@@ -0,0 +1,13 @@
+ACCOUNT_SVC_AUTH_URL="op://keycloak/$APP_ENV/base/KEYCLOAK_AUTH_TOKEN_URL"
+ACCOUNT_SVC_CLIENT_ID="op://keycloak/$APP_ENV/entity-service-account/ENTITY_SERVICE_ACCOUNT_CLIENT_ID"
+ACCOUNT_SVC_CLIENT_SECRET="op://keycloak/$APP_ENV/entity-service-account/ENTITY_SERVICE_ACCOUNT_CLIENT_SECRET"
+ACCOUNT_SVC_TIMEOUT="op://keycloak/$APP_ENV/entity-service-account/ENTITY_SERVICE_ACCOUNT_SVC_TIMEOUT"
+BUSINESS_API_URL="op://API/$APP_ENV/business-api/BUSINESS_API_URL"
+BUSINESS_API_VERSION_2="op://API/$APP_ENV/business-api/BUSINESS_API_VERSION_2"
+COLIN_API_URL="op://API/$APP_ENV/colin-api-entity/COLIN_API_URL"
+COLIN_API_VERSION="op://API/$APP_ENV/colin-api-entity/COLIN_API_VERSION"AUDIENCE="op://gcp-queue/$APP_ENV/base/AUDIENCE"
+AUDIENCE="op://gcp-queue/$APP_ENV/base/AUDIENCE"
+PUBLISHER_AUDIENCE="op://gcp-queue/$APP_ENV/base/PUBLISHER_AUDIENCE"
+GCP_AUTH_KEY="op://gcp-queue/$APP_ENV/a083gt/BUSINESS_GCP_AUTH_KEY"
```

Comment:
Right now 'BUSINESS_GCP_AUTH_KEY' is expected in the config so either need to change it here or in the config